CHANGELOG.md

```diff
@@ -7,6 +7,11 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## Unreleased
+
+* Fix bug when getting target element from URL hash in accordion component ([PR #2985](https://github.com/alphagov/govuk_publishing_components/pull/2985))
```

Comment:
Was this missed in a previous PR?

Reply:
Yeah, when I added it I was still getting a merge conflict for some reason which resulted in the strange commit at the end.

Reply:
And the order of the commits got a bit jumbled, I'll have a look at sorting it out before I merge.